init.sh

```diff
@@ -312,14 +312,14 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
-docker pull "decentraland/catalyst-content:${DOCKER_TAG:-latest}"
+docker pull "quay.io/decentraland/catalyst-content:${DOCKER_TAG:-latest}"
 if [ $? -ne 0 ]; then
   echo -n "Failed to pull the content's docker image with tag ${DOCKER_TAG:-latest}"
   printMessage failed
   exit 1
 fi
 
-docker pull "decentraland/catalyst-lambdas:${DOCKER_TAG:-latest}"
+docker pull "quay.io/decentraland/catalyst-lambdas:${DOCKER_TAG:-latest}"
```

Comment:
I think we need to include `decentraland/catalyst-lighthouse` too.

Reply:
It's a few lines below.

Reply:
> I think we need to include `decentraland/catalyst-lighthouse` too.

There are no images in `quay.io/decentraland/catalyst-lighthouse` because its pipeline is pushing to `docker.io`